### DISCUSSION TITLE
请问是否可以更改对象存储类型和存储数据的桶路径？

### DISCUSSION THREAD

**Discussion by gdtv** [juicedata/juicefs#4940] [Q&A] [answered]
例如我原来存储在本地的 /data/files 目录，我想改为存储在 webdav的 https://test.com/webdav/files 上面，我已经手动将 /data/files 里的所有文件移动到 https://test.com/webdav/files 上面了。
要怎样修改juicefs的设置呢？

**Comment by jiefenghuang** [comment]
新建一个juicefs, 用sync工具同步
https://juicefs.com/docs/zh/community/guide/sync

**Comment by gdtv** [comment, marked as answer]
自己回答一下： juicefs config 可以修改
